coffee-module/coffee-module-redisstream/src/main/java/hu/icellmobilsoft/coffee/module/redisstream/consumer/RedisStreamConsumerExecutor.java

```diff
@@ -95,6 +95,7 @@ public void init(String redisConfigKey, String group, Bean<? super IRedisStreamB
         this.redisConfigKey = redisConfigKey;
         redisStreamService.setGroup(group);
         this.consumerBean = consumerBean;
+        this.streamGroupConfig.setConfigKey(group);
```

Comment:
itt ez nekem nagyon korai beallitasnak tunik, mivel ez a z executor sok streamet kezel egyszerre, az initbe nem lehet globalisan a konfig kezelojebe beallitani barmit is